subsys/metrics/src/main/java/org/commonjava/indy/metrics/jaxrs/interceptor/MetricsInterceptor.java

```diff
@@ -0,0 +1,65 @@
+package org.commonjava.indy.metrics.jaxrs.interceptor;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.Timer;
+import org.commonjava.indy.IndyMetricsManager;
+import org.commonjava.indy.measure.annotation.IndyMetrics;
+import org.commonjava.indy.measure.annotation.Measure;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Created by xiabai on 2/27/17.
+ */
+@Interceptor
+@IndyMetrics
+public class MetricsInterceptor {
+
+    private static final Logger logger = LoggerFactory.getLogger(MetricsInterceptor.class);
+
+    @Inject
+    IndyMetricsManager util;
+
+    @AroundInvoke
+    public Object operation(InvocationContext context)
+            throws Exception {
+        logger.info("call in MeterHandler.operation");
+        IndyMetrics metrics = context.getMethod().getAnnotation(IndyMetrics.class);
```

Comment:
All of this stuff should only execute if config.isEnabled() == true. Otherwise, we should just pass through to the invocation context.